lib/req-info.js

```diff
@@ -1,8 +1,9 @@
 'use strict';
 
-module.exports = (req) => ({
+module.exports = (req) => (req && req.headers && {
```

Comment:
I think this should fail in the case the req isn't a valid request object. i.e. I'd remove the checks, `req && req.headers && `.